ipad/Utilities/APICore/RemoteAPI.swift

```diff
@@ -60,33 +74,71 @@ class RemoteAPI<R>: ExpressibleByStringLiteral {
         return ""
     }
     
+    func postBody() -> JSONCodable? {
+        return self.reqBody
+    }
+    
+    func putBody() -> JSONCodable? {
+        return self.reqBody
+    }
+    
     var isValid: Bool {
         return !self._url.isEmpty && URL(string: self._url) != nil
     }
     
+    
     var token: String {
```

Comment:
Auth.getAccessToken() returns user's token

Reply:
Currently running in test mode but will replace that logic with actual token